### PR TITLE
Task2 s4 pokemon objects

### PR DESCRIPTION
**Pull request by aabdiaziz**

### CODE REVIEW

File: src/classes/Pokemon.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<?php
+
+
```
Comment (Skylucas) [inline src/classes/Pokemon.php]: line 3 LB

File: src/classes/Pokemon.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+namespace theNamespace\classes;
+
+
```
Comment (Skylucas) [inline src/classes/Pokemon.php]: line 6 LB

File: src/classes/Pokemon.php [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+     */
+    public function __construct(\PDO $db)
+    {
+        $this->dbConnection = $db;
+    }
+
+    /**
+     * function fetches query results from database and populates class Pokemon properties.
+     *
+     * @return array of results from the query, the query generates an array of Assoc and indexed array of the the properties. name,id,type,type2
```
Comment (Skylucas) [inline src/classes/Pokemon.php]: line 28, doc block fails standard eg it goes past the line

File: dex.php [hunk 4]
```diff
@@ -1,3 +1,15 @@
+<?php
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+$db = new PDO('mysql:host=127.0.0.1; dbname=pokedex', 'root');
+
+
```
[the remark this answers is not in the document]
Comment (aabdiaziz) [reply dex.php]: done

**Pull request merged**